src/filesinkhelper.ipp

```diff
@@ -79,9 +79,7 @@ namespace g3 {
       std::string header() {
          std::ostringstream ss_entry;
          //  Day Month Date Time Year: is written as "%a %b %d %H:%M:%S %Y" and formatted output as : Wed Sep 19 08:28:16 2012
-         ss_entry << "\t\tg3log created log at: " << g3::localtime_formatted(g3::systemtime_now(), "%a %b %d %H:%M:%S %Y") << "\n";
-         ss_entry << "\t\tLOG format: [YYYY/MM/DD hh:mm:ss uuu* LEVEL FILE->FUNCTION:LINE] message";
```

Comment:
Great job. If not contrary feedback comes from the called out g3log users I am good with these changes. 


As far as I can see the only thing missing now is tiny formatting issues (CMake) and to put the header back for the beginning of the log file. 


Thanks